cares_reinforcement_learning/util/Plot.py

```diff
@@ -71,7 +71,8 @@ def post(self, reward):
             plt.pause(10e-10)
 
         if self.plot_num % self.checkpoint_freq == 0:
-            self.save_csv(f'{self.title}.csv')
```

Comment:
need to address this one

Reply:
Agreed. It's part of this #35. The Plot class was "deprecated" because it was too hard to use.